### DISCUSSION TITLE
Window PoSt Survival Guide

### DISCUSSION THREAD

**Discussion by Koru911** [filecoin-project/lotus#5659] [Tutorials]
Having personally experienced much frustration with missing window PoSt we have collated a Window PoSt Survival Guide based on our experience and much support received from fellow community members. We hope it proves useful.

[Window PoSt Survival Guide v1.pdf](https://github.com/filecoin-project/lotus/files/6025744/Window.PoSt.Survival.Guide.v1.pdf)

**Comment by s0nik42** [comment]
Thank you @Koru911 really like it ! what do you think about adding a link to the penalties calculation ? 

Also one question regarding the "lotus-miner proving check" ; let's say, one sector is corrupted and is reported as faulty during my last PoST, if I restore the file from a backup, can I run "lotus-miner proving check" before the next deadline to see if it will pas the next PoST ?

Thank you,

Julien

**Reply by Koru911** [reply]
Hey @s0nik42  thanks for the feedback - we will update the document with the feedback we are receiving. And yes - we've had recent issues with an old HDD corrupting sectors, the "lotus-miner proving check" would sometimes see a bad sector and sometimes not, so before we had chance to restore the HDD, we ran the proving check just before window PoSt and if anything was bad, removed that sector. Not ideal, but helped avoid a slash.

**Reply by s0nik42** [reply]
When you said removed the sector. You mean terminating it ?
